packages/landingpage-new/components/home/core-aspects/core-aspects.tsx

```diff
@@ -1,29 +1,47 @@
-import { InoIcon } from '@elements';
 import styles from './core-aspects.module.scss';
+import versatile from '@assets/coreaspect-versatile.svg';
+import consistent from '@assets/coreaspect-consistent.svg';
+import futureproof from '@assets/coreaspect-futureproof.svg';
+import Image from 'next/image';
 
 export default function CoreAspects() {
   return (
     <div className={styles.secondContainer}>
       <div>
```

Comment:
I also would give these its own class instead of relying on the element selector. How about `coreAspects`?